aspnetcore/migration/webapi/sample/6.x/ProductsApp/App_Start/WebApiConfig.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
```

Comment:
Delete this project. What do you need it for?

Reply:
It seemed needlessly complicated to make the reader go search for the starting project we use for this 6.x migration version of the doc.  We have this problem in a number of topics where it is not clear where all the code is we are referring to for the topic and you have to peck around.  I will get rid of it, but that is why I provided it in one place.

Reply:
It's trivial. They're experts in this version and new to Core.